strings/pirates.json

```diff
@@ -171,6 +173,18 @@
 		"The monochrome crash",
 		"Maintrastosphere Omega"
 	],
+	"irs_names":[
+		"Iron Shell Agency",
+		"Auroran Federal Tax Bureau",
+		"TerraGov Internal Revenue Service",
+		"Spinward Federal Taxation Office",
+		"Soviet Taxation Administration, 5th Service",
+		"Tiziran Imperial Tribute Service",
```

Comment:
How they gonna be tiziran if they aint lizard..... HMM

Reply:
a solid 2/3rd of the names have been starsector references I was getting desperate. I have replaced it with a homeworld reference instead